examples/skills_tuning/instructlab/annotation_classification.py

```
@@ -0,0 +1,300 @@
+#     nbconvert_exporter: python
+#     pygments_lexer: ipython3
+#     version: 3.11.12
+# ---
+
+# %%
+# %load_ext autoreload
+# %autoreload 2
+
+# %%
+# Standard
+import random
```

Flake8 (7.2.0)

[error] 'random' imported but unused

(F401)